extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -84,6 +85,20 @@ public interface HibernateOrmRuntimeConfigPersistenceUnit {
     @ConfigDocMapKey("full-property-key")
     Map<String, String> unsupportedProperties();
 
+    /**
+     * Additional database object types to include in schema management operations.
+     * <p>
+     * By default, Hibernate ORM only considers tables and sequences when performing schema management operations.
+     * This setting allows you to specify additional database object types that should be included,
+     * such as "MATERIALIZED VIEW", "VIEW", or other database-specific object types.
+     * <p>
+     * The exact supported values depend on the underlying database and dialect.
+     *
+     * @return A list of additional physical table types to be managed by Hibernate ORM
+     * @asciidoclet
+     */
+    List<String> extraPhysicalTableTypes();
```

Comment:
I think we need this, because for some reason Smallrye config won't trim configuration by default:

```suggestion
    List<@WithConverter(TrimmedStringConverter.class) String> extraPhysicalTableTypes();
```